src/gluonts/mx/model/estimator.py

```diff
@@ -151,11 +156,6 @@ def train_model(
 
         training_data_loader = self.create_training_data_loader(
             transformed_training_data
-            if not cache_data
-            else Cached(transformed_training_data),
-            num_workers=num_workers,
-            num_prefetch=num_prefetch,
-            shuffle_buffer_length=shuffle_buffer_length,
```

Comment:
These can actually be updated to use `loader_args` right?

Reply:
Just need to find where they are used.